lib/game.rb

```diff
@@ -0,0 +1,129 @@
+require 'pry'
+
+class Game
+	
+	attr_accessor :board, :player_1, :player_2
+	
+	WIN_COMBINATIONS = [
+	  [0,1,2], # Top row
+	  [3,4,5],
+	  [0,3,6],# Middle row
+	  [6,7,8],  # Bottom row
+	  [0,4,8],  # First column
+	  [1,4,7], # Second column
+	  [2,5,8], #Third column #Diagonal column
+	  [2,4,6], #Diagonal column
+	]
+	
+
+	def initialize(player_1 = Players::Human.new("X"), player_2 = Players::Human.new("O"), board = Board.new)
+		@player_1 = player_1
+		@player_2 = player_2
+		@board = board
+	end
+
+	def turn_count
+		counter = 0
+		board.cells.each do |turn|
+			if turn == "X" || turn == "O"
+				counter +=1
+			end
+		end
+		counter		
+	end
+
+	def current_player		
+		if self.turn_count.even? == TRUE
```

Comment:
You can use a ternary operator here 

`self.turn_count.even? ? : player_1 : player_2`